Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -1347,24 +1389,27 @@ theorem toL1_sub (f g : α → β) (hf : Integrable f μ) (hg : Integrable g μ)
 #align measure_theory.integrable.to_L1_sub MeasureTheory.Integrable.toL1_sub
 
 theorem norm_toL1 (f : α → β) (hf : Integrable f μ) :
-    ‖hf.toL1 f‖ = ENNReal.toReal (∫⁻ a, edist (f a) 0 ∂μ) := by simp [to_L1, snorm, snorm'];
+    ‖hf.toL1 f‖ = ENNReal.toReal (∫⁻ a, edist (f a) 0 ∂μ) := by
+  simp [toL1, snorm, snorm']
   simp [edist_eq_coe_nnnorm]
 #align measure_theory.integrable.norm_to_L1 MeasureTheory.Integrable.norm_toL1
 
 theorem norm_toL1_eq_lintegral_norm (f : α → β) (hf : Integrable f μ) :
     ‖hf.toL1 f‖ = ENNReal.toReal (∫⁻ a, ENNReal.ofReal ‖f a‖ ∂μ) := by
-  rw [norm_to_L1, lintegral_norm_eq_lintegral_edist]
+  rw [norm_toL1, lintegral_norm_eq_lintegral_edist]
 #align measure_theory.integrable.norm_to_L1_eq_lintegral_norm MeasureTheory.Integrable.norm_toL1_eq_lintegral_norm
 
 @[simp]
 theorem edist_toL1_toL1 (f g : α → β) (hf : Integrable f μ) (hg : Integrable g μ) :
     edist (hf.toL1 f) (hg.toL1 g) = ∫⁻ a, edist (f a) (g a) ∂μ := by
-  simp [integrable.to_L1, snorm, snorm']; simp [edist_eq_coe_nnnorm_sub]
+  simp [Integrable.toL1, snorm, snorm']
+  simp [edist_eq_coe_nnnorm_sub]
```

Comment:
Similarly here, and below, apparently nonterminal simps that should be combined with the subsequent simp.

Reply:
These can't because `snorm` must be used earlier than `edist_eq_coe_nnnorm`.